tests/testing_data/actions/actions.yml

```diff
@@ -222,3 +222,16 @@ pipedrive_leads_action:
   response: Thankyou for your time
   metadata:
     name: name_slot
+
+two_stage_fallback:
+- fallback_message: I could not understand you! Did you mean any of the suggestions
+    below? Or else please rephrase your question.
+  name: kairon_two_stage_fallback
+  text_recommendations:
+    count: 0
+    use_intent_ranking: true
+  timestamp: 2023-03-15 18:15:00.717000
```

Comment:
2 stage fallback has timestamp? remove it from the function that gets it and put assertion